src/rcc.rs

```diff
@@ -591,6 +613,30 @@ impl CFGR {
     }
 }
 
+/// RCC Clock Control register (RCC_CR)
+pub struct CR {
+    _0: (),
+}
+
+impl CR {
+    pub(crate) fn cr(&mut self) -> &rcc::CR {
+        // NOTE(unsafe) this proxy grants exclusive access to this register
```

Comment:
That's not true. If you look at `CFGR`'s `freeze()` method, that one also modifies the `CR` register, so the access here is not exclusive at all. I guess `CFGR` already assumes exclusive access to `CR` though that's not documented unfortunately. Which means this `CR` proxy shouldn't exist. Or at least it shouldn't exist as long as `freeze()` was not called